Tests/Fuzzers/default.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+close_fd_mask=3
```

Comment:
this is probably related to oss-fuzz? 🤔 
What does it do?

Reply:
Yes, this is a [libFuzzer flag](https://llvm.org/docs/LibFuzzer.html). It says we are not interested errors and warnings to stdout/stderr. Nothing changes here, the file is just moved from https://github.com/google/oss-fuzz/pull/11198